### DISCUSSION TITLE
Unauthenticated map/view for monitoring display

### DISCUSSION THREAD

**Discussion by SuperTrollMan** [OctoFarm/OctoFarm#1136] [Feature Request]
Please consider the addition of a customizable view (like the dashboard, but probably with some camera feeds on there) that can be accessed from a URL without authentication requirements.

This would be useful to create monitoring displays in printing labs allowing thin clients to load the page on boot without requiring peripherals for authentication to the view.

Thanks!

**Comment by NotExpectedYet** [comment]
Think I might take a stab at this. 

I may create a second dashboard just for this. With no navigation, the same widgets but separate layout. How does that sound? 

Would be under a route like /display if enabled. 

So: 
 - [ ] Add an option to enable a /display route that is accessible without authentication
 - [ ] Add a new settings page that allows an administrator to configure this page with currently available dashboard widgets
 - [ ] Create a new layout template for use without the navigation options
